packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -5,8 +5,7 @@ import 'package:flutter_driver/driver_extension.dart';
 
 void main() {
 {{#withDriverTest}}
-  // Starts the app with Flutter Driver extension enabled to allow Flutter Driver
-  // to test the app.
+  // Starts with Flutter Driver extension enabled allowing it to test the app.
```

Comment:
Maybe: "Allow the Flutter Driver to test the app. See goo.gl/i83h2f for more info."

(I made up that link. Maybe point it to https://flutter.io/testing/ ?)